core/domain/prod_validation_jobs_one_off.py

```diff
@@ -5038,6 +5039,98 @@ def _get_custom_validation_functions(cls):
         ]
 
 
+class PlaythroughModelValidator(BaseModelValidator):
+    """Class for validating PlaythroughModel."""
+
+    PLAYTHROUGH_PROJECT_RELEASE_DATETIME = datetime.datetime(2018, 9, 1)
+
+    @classmethod
+    def _get_external_id_relationships(cls, item):
+        return {
+            'exp_id': (
```

Comment:
```suggestion
            'exp_ids': (
```

Reply:
Why use plural here? This field `PlaythroughModel.exp_id` should only contain one exploration id.

Reply:
Because all other fields use plural, we should stay consistent with them. @ankita240796 should we revisit changing the names of these fields?

Reply:
Got it, done.